test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -8,4 +8,23 @@ Describe "New-PSSession basic test" -Tag @("CI") {
             $_.FullyQualifiedErrorId | Should Be "InvalidOperation,Microsoft.PowerShell.Commands.NewPSSessionCommand"
         }
     }
+}
+
+Describe "JEA session bug fix test" -Tag @("Feature", 'RequireAdminOnWindows') {
```

Comment:
I believe that we should not mention "bug fixes"  in the title and it is better to make comments in `It` blocks with a link to the bug number below.
```powershell
# Fix #1234567890
```